hi-ml-azure/src/health_azure/himl.py

```diff
@@ -704,8 +705,14 @@ def create_v2_inputs(ml_client: MLClient, input_datasets: List[DatasetConfig]) -
     inputs: Dict[str, Input] = {}
     for i, input_dataset in enumerate(input_datasets):
         input_name = f"INPUT_{i}"
-        version = input_dataset.version or 1
-        data_asset: Data = ml_client.data.get(input_dataset.name, version=str(version))
+        version = input_dataset.version
+        # if version is None, this function gets the latest version
+        data_asset: Data = _get_or_create_v2_data_asset(
```

Comment:
Can you make sure we are printing progress information here, like "now trying to access data asset foo version None in workspace bar"